tests/run-test-suite/src/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM alpine:3.4
```

Comment:
I'm curious as to whether we need yet another dockerfile hidden in `tests/run-test-suite/src/Dockerfile`. Just seems a bit ... out of place.